.github/workflows/deploy.yml

```
@@ -0,0 +1,59 @@
+on:
+  workflow_call:
+    inputs:
+      environment:
```

```suggestion
      deploy_env:
```
Should this be `deploy_env`?